### DISCUSSION TITLE
Very slow loading times on production server

### DISCUSSION THREAD

**Discussion by linkocrn** [jeffreylanters/react-unity-webgl#427] [Help Wanted!] [answered]
### Please avoid duplicates

- [X] I checked [all open issues](https://github.com/jeffreylanters/react-unity-webgl/issues) and [discussions](https://github.com/jeffreylanters/react-unity-webgl/discussions) and none of them matched my problem.

### Language and Compiler

Vanilla JavaScript

### What environment are you using?

Static File Serving

### When does your problem occur?

While the Unity App is loading

### What does your problem relate to?

The problem seems Unity related

### React-Unity-WebGL Version

9.0.3

### React Version

17.0.2

### Unity Version

2021.3.3f1

### What happened?

On my local machine, le loading is very fast and almost instantaneous.
Now I uploaded everything on a ubuntu server and everything works but the loading is now extremely slow at times (not always the same).
Are the game files not cached?
is 70 MB huge for a game?

### Would you be interested in contributing a fix?

- [ ] yes, I would like to contribute a fix

**Comment by jeffreylanters** [comment, marked as answer]
Hi. Caching is completely depending on your server's configuration. 70mb is quite big for a WebGL build, but it depends on it's purpose. I would recommend keeping it under around 30mb.

I hope this helps! Happy coding.

**Comment by Mir4nda** [comment]
I have a similar problem.
I think that could be a cache problem.... I report this: https://github.com/jeffreylanters/react-unity-webgl/discussions/436

@jeffreylanters is it possible to configure cache with your lib?

**Reply by jeffreylanters** [reply]
I'm going to look into it!

**Reply by Mir4nda** [reply]
thank you :)
